### DISCUSSION TITLE
Oled on Speedybee F405 WING

### DISCUSSION THREAD

**Discussion by zebu14** [iNavFlight/inav#9753] [Q&A]
Hello,

I have a Matek F405 Wing and a Speedybee f405 Wing.

On the Matek, i know that I can wire an oled to the I2C1 (or I2C2 eventually with the target.h modification)

I'd like to use the same oled screen with the Speedybee.
Do someone has already done it or knows if it's possible ?

It seems that there is only one I2C available on the Speedybee F405 WING, so can I use multiple peripherals on a single I2C bus (ex : magnometer and oled) or do I need to choose between them ?

I can't find any information about it so I ask here.
Thank you

**Comment by whatasteele** [comment]
Zebu14,
I came across this post looking for the same thing. Did you find your answer? if so can you post it?

**Reply by zebu14** [reply]
Hello,
Haven't tried on the speedybee but I have been advised to wire the oled screen on the same pads as the GPS, because you can have "as many i2c devices on a same bus (within reason)".

The speedybee offers both solder pads and connector for the GPS and other sensors so it should be easy to wire both oled screen and GPS.
